plugins/woocommerce/client/blocks/assets/js/data/cart/index.ts

```diff
@@ -65,11 +67,15 @@ window.addEventListener( 'load', () => {
 
 	if (
 		( ! hasCartSession() || hasItemsInCachedCart ) &&
-		! isAddingToCart()
+		! isAddingToCart() &&
+		! isEditor() // Don't finish resolution in editor,but only for real carts
 	) {
-		// Prevent the API request from being made.
 		wpDispatch( store ).finishResolution( 'getCartData' );
 	}
+
+	if ( isEditor() ) {
+		wpDispatch( store ).receiveCart( previewCart );
+	}
```

Comment:
Why is this needed? We resolve `previewCart` in `getCartData` if in the editor. I removed this and it seemed to work still?